ravedude/src/main.rs

```diff
@@ -286,6 +294,8 @@ fn ravedude() -> anyhow::Result<()> {
         },
     }?;
 
+    let console_port = ravedude_config.general_options.console_port.clone();
+
```

Comment:
Why do this up here instead of futher down the function, right next to the

```rust
let port = console_port
```

part?

Reply:
no good reason. i'll move it.